andela/octopus-bn-backend
# #170947434 user inputs validation

#### What does this PR do?
add user inputs validation feature
#### Description of Task to be completed?
- install express-validator
- added the validation middleware
- added validation middleware tests
#### How should this be manually tested?
- clone this repo
- run npm install
- set up database (see readme)
- run npm run seedDb
- run npm run dev
- access the signup endpoint(/api/v1/auth/signup) through postman
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#170947434
#### Screenshots (if appropriate)
#### Questions:- added the user route file
